examples/vr-app-n-stas.cc

```diff
@@ -168,6 +169,10 @@ main (int argc, char *argv[])
   YansWifiChannelHelper channel = YansWifiChannelHelper::Default ();
   YansWifiPhyHelper phy;
   phy.SetChannel (channel.Create ());
+  phy.Set("ChannelSettings",
+          StringValue(std::string("{0, ") + std::to_string(channelWidth) +
+                      ", " + (frequency == 2.4 ? "BAND_2_4GHZ" : "BAND_5GHZ") +
+                      ", 0}"));
```

Comment:
i would suggest to make this more robust by either checking if it is really 2.4 or 5 or even creating an enum type. right now, specifying a different frequency (e.g., 60 GHz), the simulation would run with 5GHz by default without throwing errors

Reply:
I made the example compliant only with 802.11ac (which was the selected standard for this example). As it only supports 5GHz band, I added the check on that, aborting the script if check does not pass.